deploy/crds/mattermost_v1alpha1_clusterinstallation_crd.yaml

```diff
@@ -102,6 +102,31 @@ spec:
                     is blue or green.
                   type: string
               type: object
+            canary:
+              properties:
+                canaryDeployment:
+                  description: CanaryDeployment defines the canary deployment.
+                  properties:
+                    image:
+                      description: Image defines the base Docker image that will be
+                        used for the deployment. Required when BlueGreen is enabled.
+                      type: string
+                    ingressName:
+                      description: IngressName defines the ingress name that will
+                        be used by the deployment.
+                      type: string
```

Comment:
Is there a way we can not have this appear in the CRD while keeping it part of AppDeployment? If not, let's update the comment to mention that it is not used for canary builds.

Reply:
We cannot make it not to appear but I updated the comment to specify that this option is not for Canary deployments.